docs/docsite/rst/user_guide/playbooks_intro.rst

```diff
@@ -469,29 +478,44 @@ Run ``ansible-pull --help`` for details.
 
 There's also a `clever playbook <https://github.com/ansible/ansible-examples/blob/master/language_features/ansible_pull.yml>`_ available to configure ``ansible-pull`` via a crontab from push mode.
 
-.. _tips_and_tricks:
+.. _testing_playbooks:
 
-Tips and Tricks
-```````````````
+Testing playbooks
```

Comment:
In light of @Akasurde's comment, this title should probably be "Linting playbooks" rather than "Testing playbooks"